src/core/layout/scrollable-list/scrollable-list-buttons.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @module core/layout/scrollable-list
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0 Apache-2.0
+ */
+import { onClick } from "./scrollable-list-helpers.js";
+import { eventBus } from "../../event-bus.js";
+import { overlays1Wide } from "./button-overlays.js";
+import fp from "../../../../lib/lodash/fp/fp.js";
+
+export const createGelButton = (scene, item) => {
+    const id = `shop_id_${item.id}`;
```

Comment:
Might make a note somewhere that we should probably look at this. We can't always rely on a hard-coded word like "shop" because the scene name can be changed. I don't think we need to worry about this right now though - just mentioning it incase it comes up in later shop work

Reply:
I'll make it more generic for now- perhaps later we can set it from some config value.

Reply:
ab27f58